src/NavItem.ts

```diff
@@ -0,0 +1,23 @@
+class NavItem extends HTMLElement {
+  static NAV_ITEM_NOT_FOUND_ERR = 'Nav Item Not Found';
```

Comment:
There should be no such thing as 'Nav Item Not Found.' It should be 'li not found' because the **nav-item** element has to be there. It is **its innerHTML** might not be there. Refer to the previous custom component and see what errors we have. We only have the errors for not being able to find child elements like input and form child element in **search-bar** and **answer-form**.